documents/knowledge/elements.yaml

```diff
@@ -1027,18 +1027,18 @@
 
 - element: "e"
   name: Exponentiation
-  description: Exponentiate two numbers / get every nth character of a string / get length of a regex match
+  description: Exponentiate two numbers / extend string / get length of a regex match
   arity: 2
   overloads:
     num-num: a ** b (exponentiation)
-    str-num: every bth character of a
-    num-str: every ath character of b
+    str-num: repeat characters of a until a is length b
+    num-str: repeat characters of b until b is length a
     str-str: regex.search(pattern=a, string=b).span() (Length of regex match)
   vectorise: true
   tests:
     - "[5,3] : 125"
     - "[0,0] : 1"
-    - '["hello",2] : "hlo"'
+    - '["hello",7] : "hellohe"'
```

Comment:
What happens for negative numbers? Would `["hello", -7]` become `"eholleh"`?

Reply:
considering that length can't be negative it should probably error

Reply:
Well, this is a golflang, so why not stuff some useful behavior in there instead of erroring? :P (although the useful behavior should probably make sense, I guess (which I feel reversing does))

Reply:
As I said in chat, this overload should go to `Ẏ` as it's a logical extension of its current functionality.